flight/mcu/lib/teensyShared/ActiveControl.h

```diff
@@ -1,41 +1,37 @@
 #include <PWMServo.h>
-#include <math.h>
 #include <rk4.h>
 
-#include <array>
-
 #include "ServoControl.h"
-#include "dataLog.h"
 
-using std::array;
+class Controller;
+
+extern Controller activeController;
 
 class Controller {
    public:
-    void ctrlTickFunction(pointers* pointer_struct);
+    void ctrlTickFunction();
     bool ActiveControl_ON();
-    Controller(struct pointers* pointer_struct, PWMServo* controller_servo);
+    Controller();
+    void init();
+
+    void startup();
```

Comment:
Having init, startup, and a constructor is confusing. Why is init and startup separate?

Reply:
because that was the way it was before and I didn't think to change it

Reply:
fair enough